src/audio.c

```diff
@@ -368,6 +369,23 @@ audio_get_channel(Audio *audio)
 }
 
 /**
+ * Whether the card has mute capabilities.
+ *
+ * @param audio an Audio instance.
+ * @return TRUE if the card can be muted, FALSE otherwise.
+ */
+gboolean
+audio_has_mute(Audio *audio)
```

Comment:
is it possible that `*audio` is NULL? In that case, we'd get NULL dereference in line 380